src/Core/src/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -140,5 +126,46 @@ public void Configure(HostBuilderContext context, IServiceProvider services)
 			{
 			}
 		}
+
+		class HotReloadBuilder : IMauiServiceBuilder
+		{
+			public string? IdeIp { get; set; }
+
+			public int IdePort { get; set; } = 9988;
+
+			public void ConfigureServices(HostBuilderContext context, IServiceCollection services)
+			{
+			}
+
+			public async void Configure(HostBuilderContext context, IServiceProvider services)
+			{
+				var handlers = services.GetRequiredService<IMauiHandlersServiceProvider>();
```

Comment:
This method runs _after_ the services build has completed and thus we get access to all services and can ensure that Reloadify doesn't accidentally do something with a half-completed build.